vignettes/bds_tte.Rmd

```diff
@@ -430,12 +431,12 @@ adsl <- adsl_bak
 # define censoring #
 observation_end <- censor_source(
   dataset_name = "adsl",
-  date = EOSDT,
+  date = pmin(TRTEDT + days(30), EOSDT),
```

Comment:
is `pmin` a base function or come from another package?

Reply:
`pmin()` is a base function.